actors/miner/tests/util.rs

```diff
@@ -1183,6 +1198,27 @@ pub fn assert_bitfield_equals(bf: &BitField, bits: &[u64]) {
     assert!(bf == &rbf);
 }
 
+#[allow(dead_code)]
+pub fn make_empty_bitfield() -> UnvalidatedBitField {
+    UnvalidatedBitField::Validated(BitField::new())
+}
+
+#[allow(dead_code)]
+pub fn make_bitfield(bits: &[u64]) -> UnvalidatedBitField {
+    let mut bf = BitField::new();
+    for bit in bits {
+        bf.set(*bit);
+    }
+    UnvalidatedBitField::Validated(bf)
```

Comment:
```suggestion
    UnvalidatedBitField::Validated(bits.iter().copied().collect())
```

Reply:
ok, that's significantly shorter even if not as readable.

Reply:
applied your magic incantation.

why do we need the `copied` spell?